core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -584,6 +585,11 @@ protected void onModified() throws IOException {
      */
     /*package*/ int scmCheckoutRetryCount;
 
+    /**
+     * For {@link hudson.slaves.ComputerRetentionWork#getRecurrencePeriod()}
+     */
+    /*package*/ int computerRetentionCheckInterval = 60;
```

Comment:
Why is this added to `Jenkins`, instead of the new `GlobalConfiguration`?

Reply:
Why should it be added to the new `GlobalConfiguration`?

Reply:
Ideally we'd not make Jenkins even more of a god object. This seems like a reasonable option to store elsewhere, not unlike `JenkinsLocationConfiguration`.

Reply:
I adapted the storage from the `quietPeriod`/`scmRetryCount` config options, which both store their value in the `Jenkins` instance.
I moved the value into the model via dab113783ecfdf84685bcf8ead4c178c5fdc6c5f and repeated manual testing for (de-)serialization/updating via the GUI.